### PR TITLE
Autofixer for newline-after-import. Fixes #686

### LINKED ISSUE #686: Add autofixer for newline-after-import
Per title.
The fix is super simple.

### PR DESCRIPTION
**Pull request by eelyafi**
A PR for https://github.com/benmosher/eslint-plugin-import/issues/686

Will add autofixer for newline-after-import rule.

### CODE REVIEW

**Comment by eelyafi** [comment]
Keep having this error locally. Can repro on master branch as well. Does anyone know what to do ?
```
1 failing

  1) package has every rule:
     Uncaught AssertionError: expected { Object (no-unresolved, named, ...) } to have a property 'npm-debug.log'
      at tests/src/package.js:27:14
      at Array.forEach (native)
      at tests/src/package.js:25:15
      at FSReqWrap.oncomplete (fs.js:117:15)
```

**Comment by ljharb** [comment]
@eelyafi i'm quite sure that's a bug in the test; i'll fix it on master momentarily.

**Comment by ljharb** [comment]
@eelyafi actually, are you using `npm test`, or manually running mocha tests for the one file?

**Comment by eelyafi** [comment]
@ljharb I'm using `npm run test`

**Comment by ljharb** [comment]
@eelyafi and is your current working directory the root of the project? or is it inside `rules`?

**Comment by eelyafi** [comment]
also, after running `npm run test` there were no linter errors. The errors showed up only after submitting the PR

**Comment by ljharb** [comment]
I was able to reproduce the error by running `touch src/rules/npm-debug.log && npm test` - but that should only happen otherwise if you were running an npm command not at the root of the repo (all commands should always only be run at the root of the repo). You can `rm` the `npm-debug.log` for now to fix it, and I'll push something up to master to make the test more robust.

**Comment by eelyafi** [comment]
@ljharb it helped ! thanks :)

**Comment by eelyafi** [comment]
@ljharb looks like the PR passed all tests but there is no reviewers. How to add one ?

File: src/rules/newline-after-import.js [hunk 1]
```diff
@@ -42,9 +42,12 @@ function isClassWithDecorator(node) {
   return node.type === 'ClassDeclaration' && node.decorators && node.decorators.length
 }
 
+const EXPECTED_LINE_DIFFERENCE = 2
+
 module.exports = {
   meta: {
     docs: {},
+    fixable: 'code',
```
Comment (stas-vilchik) [inline src/rules/newline-after-import.js]: Similar [`newline-after-var` rule](https://github.com/eslint/eslint/blob/master/lib/rules/newline-after-var.js#L26) in eslint uses the `whitespace` option. I haven't found the difference between two options in the docs, but just for consistency I'd suggest to use `whitespace`.

**Comment by grumd** [comment]
any plans on merging this?

**Comment by jfmengels** [comment]
I'm sorry @eelyafi, I forgot about this PR, and I merged https://github.com/benmosher/eslint-plugin-import/pull/742 before I merged this one, which allows the customization of the number of lines after the imports.
Do you feel like solving the conflicts and fixing the case where that number is customized? Otherwise I'll be happy to take over. Let me know.
And obvisouly, thanks for the PR, it looks great too :)

(sorry again... :disappointed:)

**Comment by ljharb** [comment]
(This PR is now freshly rebased; thanks @truckingsim for the nudge)

**Comment by truckingsim** [comment]
Thanks for letting me know about the rebasing didn't think about that.  Appreciate the quick response!

**Comment by ljharb** [comment]
@benmosher it looks like appveyor might be failing on master (it's failing on a `no-named-default` test which isn't touched by this PR)

**Comment by ljharb** [comment]
Let's give this a day or two for extra reviews - after that, I'll plan to merge it.

**Comment by benmosher** [comment]
published with v2.5.0. thanks for the (months! of) work and attention on this, everyone! 😁

**Pull request merged**